solr/conf/solrconfig.xml

```diff
@@ -76,7 +76,7 @@
   </admin>
 
 
-  <requestHandler name="search" class="solr.SearchHandler" default="true">
+  <requestHandler name="/search" class="solr.SearchHandler" default="true">
```

Comment:
think we just must have typo'd not including the leading slash? It's needed

Reply:
good catch, idk how this was OK without the `/` 🤔

Reply:
crazy right? I think the whole `default` thing really saved us. When you do a search and tail solr, you'll see the `select` handler is used, but, this `search` one is actually used. It is bizarre and "working by accident" I think. 🙃

Reply:
woww 😬

Reply:
:nods: